app/controllers/SearchForApplicantsController.scala

```diff
@@ -41,71 +43,36 @@ trait SearchForApplicantsController extends BaseController {
   val appRepository: GeneralApplicationRepository
   val psRepository: PersonalDetailsRepository
   val cdRepository: ContactDetailsRepository
+  val searchForApplicantService: SearchForApplicantService
 
-  val MAX_RESULTS = 10
+  val MAX_RESULTS = 25
 
   def findById(userId: String, frameworkId: String) = Action.async { implicit request =>
```

Comment:
I know... out of scope so you can ignore it. But if you have a chance, would you mind to add the return type to these methods?